File: Lua/1012.lua
function split(s, delimiter)
    result = {};
    for match in (s..delimiter):gmatch("(.-)"..delimiter) do
        table.insert(result, match);
    end
    return result;
end

line = split(io.read(), ' ')
a = tonumber(line[1])
b = tonumber(line[2])
c = tonumber(line[3])

area_a = a*c/2
area_b = 3.14159 * c ^ 2
area_c = (a + b) * c/2 
area_d = b ^ 2
area_e = a*b

print('TRIANGULO: ' .. string.format("%0.3f", area_a))
print('CIRCULO: ' .. string.format("%0.3f", area_b))
print('TRAPEZIO: ' .. string.format("%0.3f", area_c))
print('QUADRADO: ' .. string.format("%0.3f", area_d))
print('RETANGULO: ' .. string.format("%0.3f", area_e))

File: C/1012.c
#include <stdio.h>
 
int main() { 
    double a, b, c, area_a, area_b, area_c, area_d, area_e;
    
    scanf("%lf %lf %lf", &a, &b, &c);

    area_a = a*c/2;
    area_b = 3.14159 * c * c;
    area_c = (a + b) * c/2;
    area_d = b * b;
    area_e = a*b;

    printf("TRIANGULO: %.3lf\n", area_a);
    printf("CIRCULO: %.3lf\n", area_b);
    printf("TRAPEZIO: %.3lf\n", area_c);
    printf("QUADRADO: %.3lf\n", area_d);
    printf("RETANGULO: %.3lf\n", area_e);

    return 0;
}